examples/inverse/plot_lcmv_beamformer.py

```diff
@@ -69,8 +69,10 @@
 colors = ['b', 'k', 'r']
 
 for pick_ori, name, desc, color in zip(pick_oris, names, descriptions, colors):
+    # compute unit-noise-gain beamformer with whitening of the leadfield and
+    # data (enabled by passing a noise covariance matrix)
     stc = lcmv(evoked, forward, noise_cov, data_cov, reg=0.05,
-               pick_ori=pick_ori)
+               pick_ori=pick_ori, weight_norm='unit-noise-gain')
```

Comment:
max-power and unit-noise-gain should be fine. I think the weird activation in the brain plot stems from an interaction of the orientation selection and restricted sensor coverage (note that the mean within the auditory cortex still looks like before). It indeed looks better when using all channels:

![max-ori_all-channels](https://user-images.githubusercontent.com/20129119/27841837-ca4d0db4-6104-11e7-8122-9059408607a5.png)